app/src/main/AndroidManifest.xml

```diff
@@ -81,7 +81,9 @@
 
         <activity android:name=".activity.AccelerometerActivity"/>
 
-        <activity android:name=".activity.MapsActivity" />
```

Comment:
@yatri1609 Please undo this change as the manifest should contain the maps activity